Content.Shared/Weapons/Ranged/Components/BallisticAmmoProviderComponent.cs

```diff
@@ -55,7 +55,6 @@ public sealed partial class BallisticAmmoProviderComponent : Component
     /// Is the magazine allowed to be cycled
     /// </summary>
     [ViewVariables(VVAccess.ReadWrite), DataField("cycleable")]
-    [AutoNetworkedField]
     public bool Cycleable = true;
```

Comment:
Don't remove networked fields to get around mispredicts.

Reply:
I removed it because I thought I previously added it while it was not needed.